### PR TITLE
Fix try-it page styling issues

### PR DESCRIPTION
**Pull request by DimalChandrasiri**

### CODE REVIEW

**Comment by keizer619** [comment]
@DimalChandrasiri 
commit messages are in standard format

File: composer/modules/web/src/plugins/try-it/views/http-client.jsx [hunk 1]
```diff
@@ -805,153 +827,192 @@ class HttpClient extends React.Component {
...
+                                                            <Form.Group width={16}>
+                                                                {requestHeaders}
+                                                            </Form.Group>
+                                                        </div>
+                                                    </Form.Field>
+                                                </Form.Group>
+                                                <Form.Group>
+                                                    <Form.Field className='http-client-body-wrapper'>
+                                                        <label>Body</label>
+                                                        <hr />
```
Comment (srsharon) [inline composer/modules/web/src/plugins/try-it/views/http-client.jsx]: Shouldn't this be divider?

File: composer/modules/web/src/plugins/try-it/views/http-client.jsx [hunk 2]
```diff
@@ -805,153 +827,192 @@ class HttpClient extends React.Component {
...
+                                            <hr />
+                                            <Form>
+                                                <Form.Group>
+                                                    <Form.Field className='http-client-response-attributes'>
+                                                        <strong>Request URL :
+                                                            <span className={cn('attribute-value', this.getStatusCodeClass(this.state.responseCode))}>
+                                                                {this.state.requestUrl}
+                                                            </span>
+                                                        </strong>
+                                                        <br />
```
Comment (srsharon) [inline composer/modules/web/src/plugins/try-it/views/http-client.jsx]: Shall we get rid of the br tags?

File: composer/modules/web/src/plugins/try-it/views/http-client.scss [hunk 3]
```diff
@@ -112,33 +112,178 @@
...
+                    background: #3b3b3b;
+                    padding: 4px;
+                    border: 1px solid black;
+                    border-radius: 5px;
+                    font-size: 10px;
+                }
+            }
+        }
+    }
+    input[type=text].header-input{
```
Comment (srsharon) [inline composer/modules/web/src/plugins/try-it/views/http-client.scss]: Can we reuse the input field from ballerina-theme..

**Pull request merged**